src/library-authoring/studio-header-wrapper/ContentTitleBlock.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+function ContentTitleBlock({ title, subtitle, destination, ariaLabel }) {
+  return (
+    <Link
+      className='content-title-block'
+      to={destination}
+      aria-label={ariaLabel}
```

Comment:
Thanks for sticking with a11y. 👍🏼 

(Disclaimer: I'm not doing a proper accessibility review here.  This is going to be the topic of a separate discussion.  Simply using more Paragon helps a lot, though.)